test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1

```diff
@@ -4298,7 +4300,7 @@ Describe 'Invoke-WebRequest and Invoke-RestMethod support Cancellation through C
 
     It 'Invoke-RestMethod: CTRL-C after stalled JSON download processes JSON response' {
         $uri = Get-WebListenerUrl -test Stall -TestValue '1/application%2fjson'
-        $result = RunWithCancellation -Command 'Invoke-RestMethod' -Uri $uri -DelayMs 1200 -WillComplete
+        $result = RunWithCancellation -Command 'Invoke-RestMethod' -Uri $uri -DelayMs 5000 -WillComplete
```

Comment:
Maybe just use the default timeout after you change it to 10 seconds above?

Reply:
I can do this, but will make all the tests quite slow except for those that don't actually CTRL-C. The delay/timeout is how long to wait to press CTRL-C after starting the download and in most cases the download is stalled so it will always wait at least the delay. Perhaps the parameter should be called `-StopAfterDelayMs` or `-CtrlCAfterDelayMs` to make this clear?
```
    [+] Invoke-WebRequest: CTRL-C Cancels request before request headers received 225ms
    [+] Invoke-WebRequest: CTRL-C Cancels request after request headers received 10.03s
    [+] Invoke-WebRequest: HTTPS CTRL-C Cancels request after request headers 10.03s
    [+] Invoke-WebRequest: Compression CTRL-C Cancels request after request headers 30.08s
    [+] Invoke-WebRequest: HTTPS with compression CTRL-C Cancels request after request headers 30.06s
    [+] Invoke-WebRequest: CTRL-C Cancels file download request after request headers received 10.04s
    [+] Invoke-WebRequest: CTRL-C after stalled file download completes gives entire file 1.05s
    [+] Invoke-RestMethod: CTRL-C Cancels request before request headers received 24ms
    [+] Invoke-RestMethod: CTRL-C Cancels request after JSON request headers received 10.02s
    [+] Invoke-RestMethod: CTRL-C after stalled JSON download processes JSON response 1.1s
    [+] Invoke-RestMethod: CTRL-C Cancels request after plain request headers received 10.04s
    [+] Invoke-RestMethod: CTRL-C after stalled atom feed download processes atom response 1.05s
    [+] Invoke-RestMethod: CTRL-C Cancels request in XML atom processing 10.03s
```

Reply:
Please ignore my comment about the "10 seconds". I misunderstood the intention. Please also revert the changes related to this comment and the "10 seconds" comment above.

Reply:
I have reduced to 5 seconds delay before the stop is sent in most cases. Have reduced to 1 second where we don't need/expect to have received a response yet before we stop. I also renamed the parameter to make it clearer what it's for.